shell/platform/embedder/embedder.h

```diff
@@ -72,6 +72,7 @@ typedef enum {
 typedef enum {
   kOpenGL,
   kSoftware,
+  kMetal,
```

Comment:
A note here to say that this is only available on Darwin platforms. Along with a a note about the relevant platform versions.